yarn.lock

```diff
@@ -16276,8 +16281,6 @@ strip-ansi@^4.0.0:
   version "4.0.0"
   resolved "https://registry.yarnpkg.com/strip-ansi/-/strip-ansi-4.0.0.tgz#a8479022eb1ac368a871389b635262c505ee368f"
   integrity sha1-qEeQIusaw2iocTibY1JixQXuNo8=
-  dependencies:
```

Comment:
Still defined as a `dependency` for `strip-ansi`:

see: https://github.com/chalk/strip-ansi/blob/v4.0.0/package.json#L46

Reply:
I don't get why `yarn` keeps on removing and adding it from PR to PR 🤔

Reply:
Probably related to multiple versions of Yarn. I will merge it and keep a closer look.